concepts/05 Widgets/Form/01 Getting Started with Form/20 Populate the Form Fields.md

```diff
@@ -0,0 +1,124 @@
+To populate form fields, assign an object to the [formData](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#formData) property. Each data field in this object produces a [simple item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/)&mdash;a label-editor pair in which the label displays the data field name and the editor edits the data field value.
+
+The Form chooses default editors based on the field's data type: [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/) for string values, [NumberBox](/Documentation/ApiReference/UI_Widgets/dxNumberBox/) for numbers, [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) for dates. You can use the [editorType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorType) property to specify an editor explicitly. In the following example, the Form uses default editors:
```

Comment:
Note: "chooses" depends on a data field **value**. dxForm will create TextBoxes if you pass an object with 'null/undefined' values in properties (or no properties). A am not sure that we should mention it at this stage but have to add a note about it.

Reply:
In my opinion, Getting Started is not the right place to note about such occasions. I think that if we don't have such a note in our docs, it's better to create a docissue.

@RomanTsukanov , what do you think?